devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy

```diff
@@ -0,0 +1,157 @@
+ /*
+  *  Copyright 2018 SmartThings
```

Comment:
you can change this too if you want

Reply:
Eh. The file I copied 98% of this from is 2018. 🤷‍♂️